frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -16,40 +25,66 @@ import { searchIndexes } from "./utils";
 
 const hotKey = "/";
 
-const Container = styled(Box)`
-  ${({ theme }) => `
-  border-radius: ${theme.shape.borderRadius}px;
-  margin-left: 0px;
-  margin-right: ${theme.spacing(2)}px;
-  background-color: ${fade(theme.palette.primary.main, 0.15)};
-  &:hover {
-    background-color: ${fade(theme.palette.primary.main, 0.25)};
-  }
-  `}
-`;
-
-const InputField = styled(TextField)`
-  max-width: 300px;
-  min-width: 300px;
-  @media screen and (max-width: 800px) {
-    min-width: 125px;
-  }
-`;
-
-const InputIcon = styled(SearchIcon)`
-  margin-top: -3px;
-`;
-
-const ResultLabel = styled(Typography)`
-  ${({ theme }) => `
-  color: ${theme.palette.primary.main};
-  font-weight: 400;
-  `}
-`;
+const Container = styled(Box)({
+  borderRadius: "4px",
+  marginRight: "8px",
+  backgroundColor: "#ffffff",
+});
+
+const InputField = styled(TextField)({
+  maxWidth: "504px",
+  minWidth: "504px",
+  "@media screen and (max-width: 800px)": {
+    minWidth: "125px",
+  },
+  ".MuiAutocomplete-input": {
+    color: "#0d1030",
+  },
+});
+
+const ResultLabel = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+});
+
+const SearchIconButton = styled(IconButton)({
+  color: "#ffffff",
+  padding: "8px",
+  "&:hover": {
+    background: "#2d3db4",
+  },
+  "&:active": {
+    background: "#2938a5",
+  },
+});
+
+const StyledCloseIcon = styled(Icon)({
```

Comment:
asked for the hover state for the close icon

Reply:
noticed that the close icon is not aligned in the middle of the input field - working on the fix